### ISSUE TITLE
Request: Touch Gesture Support

### ISSUE THREAD

**Issue by KBSmith** [archetyped/simple-lightbox#302]
## Description of feature/request

Enable gestures on touch-enabled devices for navigating through items in the lightbox.
## If you added this feature, I would:

b
a) Eat my hat
b) Jump for joy
c) Shave my cat
d) All of the above

---

**Note:** Development is prioritized by demand. Add your vote by clicking the 👍 in the bottom corner of this post if you would like to see this functionality.

**Comment by dadra** [comment]
Yes please!  This is my go to plugin and adding swipe support would be great.

**Comment by quantumleap33** [comment]
I was wondering if this was still on the list?
